.github/workflows/continuous_deployment.yml

```
@@ -32,4 +33,15 @@ jobs:
             docker image prune -f
+
+      - name: Debug environment
+        run: |
+          pwd
+          echo "Current directory contents:"
+          ls -la
+          echo "Checking if file exists:"
+          test -f docker-compose.dev.yml && echo "File exists" || echo "File does not exist"
+          echo "Current branch:"
+          git branch --show-current
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)